### PR TITLE
Add 20240917_v0.10.12_src15_dec_min_2924_tuned

### PR DESCRIPTION
**Pull request by nikoliclisa**
# New Prod config

Self check-list:

- [x] I have checked the lstchain config, in particular for:
  - [x] `az_tel` instead of `sin_az_tel` if data to be analyzed have been produced with lstchain <= v0.9.7
  - [x] "increase_nsb" and "increase_psf" are provided in "image_modifier" (if used)
- [x] I have checked the environment in the lstmcpipe config and it is the one used to analyse DL>1 data
- [x] I have provided the command (in README), or script (in additionnal .py file) used to produce the lstmcpipe config

### CODE REVIEW

**Comment by nikoliclisa** [comment]
Hi @vuillaut 
I have modified the branch 20240917_v0.10.12_src15_dec_min_2924_tuned with updated(corrected) files. Could you please check it and let me know if everything is fine.
Thank you.

**Comment by nikoliclisa** [comment]
Hi @vuillaut 
I just wanted to make sure you saw the recent updates. Let me know if anything else needs to be modified.

**Comment by nikoliclisa** [comment]
@vuillaut I have rechecked and updated the lstmcpipe config (yaml) file but I am getting the error '! Subdirectories with the same PROD_ID and analysed the same day will be overwritten !' in the CI tests. Could you please check and let me know of the error?
Thank you

**Comment by vuillaut** [comment]
> @vuillaut I have rechecked and updated the lstmcpipe config (yaml) file but I am getting the error '! Subdirectories with the same PROD_ID and analysed the same day will be overwritten !' in the CI tests. Could you please check and let me know of the error? Thank you

Hi.
That's not actually the error your are getting.
The error is:
```
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 307 column 1 (char 8273)
```
showing that your JSON file (so the lstchain config) is not correct

File: production_configs/20240917_v0.10.12_src15_dec_min_2924_tuned/README.md [hunk 1]
```diff
@@ -3,7 +3,11 @@
 I need this production to analyse the LST-1 data of src15 (see https://www.lst1.iac.es/wiki/index.php/MC_analysis_and_IRF_production#lstmcpipe_productions) with dec 29.24 deg. The median of the NSB standard deviation for this dataset is approximately 2.308 p.e.
 
 '''
+<<<<<<< HEAD
+lstmcpipe_generate_config PathConfigAllSkyFullDL1ab --dec_list dec_min_2924 --prod_id 20240917_v0.10.12_src15_dec_min_2924_tuned --kwargs source_prod_id=20240131_allsky_v0.10.5_all_dec_base
+=======
 lstmcpipe_generate_config PathConfigAllSkyFullDL1ab --dec_list dec_min_2924 --prod_id 20240917_v0.10.12_src15_dec_min_2924_tuned --kwargs source_prod_id=20240917_v0.10.12_src15_dec_min_2924_tuned
+>>>>>>> 8c85c8a6cd11e8bbaf70eebfbb9ae4ab9215c1ed
```
Comment (vuillaut) [inline production_configs/20240917_v0.10.12_src15_dec_min_2924_tuned/README.md:10]: you need to handle the git merge here by keeping only the part you want (here the first one)

File: production_configs/20240917_v0.10.12_src15_dec_min_2924_tuned/lstchain_config_2024-09-17.json [hunk 2]
```diff
@@ -304,13 +304,16 @@
...
+<<<<<<< HEAD
+=======
     "image_modifier": {
     "increase_nsb": true,
     "extra_noise_in_dim_pixels": 3.843,
     "extra_bias_in_dim_pixels": 1.07,
     "transition_charge": 8,
     "extra_noise_in_bright_pixels": 5.178
   },
+>>>>>>> 8c85c8a6cd11e8bbaf70eebfbb9ae4ab9215c1ed
```
Comment (vuillaut) [inline production_configs/20240917_v0.10.12_src15_dec_min_2924_tuned/lstchain_config_2024-09-17.json:316]: same here
the symbols `<<<<`, `====` and `>>>>` come from the git merge that you did not solve

Comment (vuillaut) [reply production_configs/20240917_v0.10.12_src15_dec_min_2924_tuned/lstchain_config_2024-09-17.json:316]: This is what makes the CI fail btw.

**Review by vuillaut — CHANGES_REQUESTED** [review]

File: production_configs/20240917_v0.10.12_src15_dec_min_2924_tuned/README.md (hunk 1 above)
Comment (nikoliclisa) [reply production_configs/20240917_v0.10.12_src15_dec_min_2924_tuned/README.md]: Ok, thank you

File: production_configs/20240917_v0.10.12_src15_dec_min_2924_tuned/lstchain_config_2024-09-17.json (hunk 2 above)
Comment (nikoliclisa) [reply production_configs/20240917_v0.10.12_src15_dec_min_2924_tuned/lstchain_config_2024-09-17.json]: Ok, I solved them by editing the json file myself, now it should be fine

**Review by vuillaut — APPROVED** [review]

**Comment by vuillaut** [comment]
Hi @nikoliclisa 
The production is done !
You may find models at `/fefs/aswg/data/models/AllSky/20240917_v0.10.12_src15_dec_min_2924_tuned/dec_min_2924/`
and dl2 data at `/fefs/aswg/data/mc/DL2/AllSky/20240917_v0.10.12_src15_dec_min_2924_tuned/TestingDataset/dec_min_2924/`

**Pull request merged**